src/main/java/org/cristalise/kernel/entity/ItemImplementation.java

```diff
@@ -109,31 +115,58 @@ public void initialise(SystemKey agentId, String propString, String initWfString
             PropertyArrayList props = (PropertyArrayList) Gateway.getMarshaller().unmarshall(propString);
             for (Property thisProp : props.list) mStorage.put(mItemPath, thisProp, locker);
         }
-        catch (Throwable ex) {
+        catch (Exception ex) {
             Logger.msg(8, "ItemImplementation::initialise(" + mItemPath + ") - Properties were invalid: " + propString);
             Logger.error(ex);
             mStorage.abort(locker);
             throw new InvalidDataException("Properties were invalid");
         }
 
-        // Store an event and the initial properties
+        History hist = new History(mItemPath, locker);
+
+        // Store an event and the outcome for initial properties

```

Comment:
Don't understand what is "outcome for initial properties"

Reply:
See line 130, the propString is an XML and store as outcome

Reply:
Please rephrase:
```
// Store the "Initialize" event and the outcome containing the initial values for properties
```